src/RangeSlider.xml

```diff
@@ -81,9 +81,9 @@
             <description>Number of decimal places for marker values</description>
         </property>
         <property key="bootstrapStyle" type="enumeration" defaultValue="primary">
-            <caption>Bootstrap style</caption>
+            <caption>Slider style</caption>
             <category>Appearance</category>
-            <description>Bootstrap color to apply to the range slider</description>
+            <description>Color to apply to the range slider</description>
             <enumerationValues>
                 <enumerationValue key="primary">Primary</enumerationValue>
                 <enumerationValue key="inverse">Inverse</enumerationValue>
```

Comment:
The default color happens to be the color used by Lib when slider is disabled or has no context... This might be confusing..